modules/elasticsearch/data-nodes.tf

```diff
@@ -3,9 +3,10 @@
  *
  * Just as with master nodes, each data node has a persistent EBS volume created for.
  * Data nodes also function as coordinators, thus each one has 9200 port open for the API.
- * Elastic load balancer is created for elasticsearch API enpoint. Every node is managed
- * by an auto scaling group, so if a helath check fails for an instance it will be
- * torn down and a new one deployed with the same EBS volume mounted.
+ * Elastic load balancer is created for elasticsearch API enpoint, unless ALBs are supplied
+ * as arguments. Every node is managed by an autoscaling group, so if a helath check fails
```

Comment:
`as arguments.`  looks like it goes with the previous line.

Reply:
You are absolutely right, it does go with the previous line. This is what we call wrap around :)
I am not really following what you think the problem is here, perhaps you are thinking of those as bullet points? If that's what you mean, I was just using convention that you use in other modules, where each line in a doc is prepended by `*`, which is not the same as bullet points.

Reply:
LOL, you are right, I misread the `* ` as bullet points (3am amnesia).